src/Components/Home/filterProfileType.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Dropdown } from 'primereact/dropdown'
+
+export default function ProfileTypeFilter({ profileType, typeHandler }) {
+  const profileTypes = [
+    { label: 'Personal', value: 'personal' },
+    { label: 'All', value: 'all' },
+    { label: 'Community', value: 'community' },
```

Comment:
I would prefer to make the selection in alphabetical order. 

![image](https://user-images.githubusercontent.com/30869493/148733347-0d33533d-ef23-47ca-8ebf-59ed08c4af91.jpeg)

```suggestion
    { label: 'All', value: 'all' },
    { label: 'Community', value: 'community' },
    { label: 'Personal', value: 'personal' }
```

Reply:
![firefox_h8Ha0XXI8K](https://user-images.githubusercontent.com/91387097/148775766-73ecc920-5513-4f4a-bd33-ecf8ef97f187.png)

I think length order will look more neat ,what you say?

Reply:
I would stick to alphabetical but I leave it for discussion.
But I think we can convert this to a new issue and merge your changes.